settings/ticket.rst

```diff
@@ -163,6 +163,44 @@ the desired agents in the ``Exception Users`` list. The search function in this
 area is only effective, if you have too many agents to display at once to help
 you with the configuration.
 
+Language Detection
+------------------
+
+Zammad offers a built in language detection for articles. By default, it is
+turned off. Turn it on by selecting a backend service and confirm by clicking
+**Submit**.
+
+After enabling it, Zammad tries to detect the language of each **new** article.
+The language is not detected for existing articles.
+
+.. note::
+   - Currently, there is only one backend service:
+     `Compact Language Detection <https://github.com/jtoy/cld>`_.
+   - Be aware that this feature is in its early stages. Enhancements may come in
+     very soon.
+   - While we noticed it works pretty well, it may fail in some edge cases.
+   - The language selection in conditions and the output in the article detail
+     view is always in english. It is currently not possible to list each
+     language in their own language.
```

Comment:
Instead of this last sentence I would just say that the language names are not translated into user's current language.